### PR TITLE
Pack of fixes 

### LINKED ISSUE #367: [Bug]: Duplicated stringtable entry: "STR_A3AP_setupFactionsTab_ldf"
### Describe the bug

When trying to use the tool Tabler to check and correct the spanish translation, it detected that duplicated string error on the file \A3-Antistasi-Ultimate\A3A\addons\scrt\Stringtable.xml:

### How to reproduce

Find that string on the document to check it is actually duplicated

### Version

11.2.1

### Have you altered the code?

No

### What i have changed

_No response_

### Map

Any map

### What server?

Private dedicated server

### Time bug occured (Server time/UTC)

_No response_

### Mods

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by wersal454**
## What type of PR is this?
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:

no driver issue fix - gamebreaking, if vehicle doubled in main faction(s) and in Rivals, game couldn't decide which driver to spawn, it was breaking QRFs and skirmishes
tags error fix - not gamebreaking(probably), hopefully fixed it without breaking anything
wind param fix - not gamebreaking, but was annoying to look at
vehicle don't fit in tent hangar "fix" - in case vehicle wasn't moved in "large" category or it's a UAV or smth, I decided to make a fix that will deal with this problem in most(if not all) cases
duplicate string for ldf fix - for #367

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #367!"

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [ ] Yes (Please provide further detail below.)

********************************************************

### CODE REVIEW

File: A3A/addons/core/functions/CREATE/fn_RivalsCreateUnit.sqf
Comment (Silence-112) [inline A3A/addons/core/functions/CREATE/fn_RivalsCreateUnit.sqf:1]: What does this do differently to the normal createUnit function?

File: A3A/addons/core/functions/CREATE/fn_createAIAirplane.sqf [hunk 1]
```diff
@@ -298,19 +296,34 @@ if (!_busy) then {
...
+						deleteVehicle _veh;
+						_veh = createVehicle [_typeVehX, (_spawnParameter select 0), [], 0, "CAN_COLLIDE"];
+						_veh setDir (_spawnParameter select 1);
+						_veh allowDamage false;
+						_veh enableSimulation false;
+						sleep 0.5;
+						_veh enableSimulation true;
+						_veh allowDamage true;
+					};
+				/* }; */
```
Comment (Silence-112) [inline A3A/addons/core/functions/CREATE/fn_createAIAirplane.sqf:323]: What does removing the isNil achieve here?

**Review by Silence-112 — COMMENTED** [review]
What do the new RivalsCreateXYZ functions do differently to the non-rival ones? Why are they needed

File: A3A/addons/core/functions/CREATE/fn_createAIAirplane.sqf (hunk 1 above)
Comment (wersal454) [reply A3A/addons/core/functions/CREATE/fn_createAIAirplane.sqf:323]: Tested with isNil and without, I don't know what isNil is even doing there, but it did prevent sleep from working

**Comment by wersal454** [comment]
> What do the new RivalsCreateXYZ functions do differently to the non-rival ones? Why are they needed

"no driver fix" isn't perfect. Probably could move stuff around, reduce amount of functions, but at least it works.

And about why it's needed, well... it lesense amount of stuff that can go wrong?

It's just easier to read and understand those functions without thousand parameters going in and out.
Oh and initVehClassToCrew is just a mess, so there it reeeeally was easier to just make a separate function for rivals

**Comment by Silence-112** [comment]
Looks fine, needs to be tested

**Pull request merged**